packages/design-system/src/components/web-components/ds-spinner/ds-spinner.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-spinner';
+
+export default {
+  title: 'Web Components/Spinner',
+  argTypes: {
+    'aria-valuetext': {
+      control: 'text',
+      description: 'The text announced to screen readers.',
+    },
+    'class-name': {
+      control: 'text',
+      description:
+        'Additional classes to be added to the spinner element. Useful for adding utility classes.',
+    },
+    inversed: {
+      control: 'boolean',
+      description: 'Set to "true" to apply the "inverse" theme.',
+    },
+    filled: {
+      control: 'boolean',
+      description: 'Adds a background behind the spinner for extra contrast.',
+    },
```

Comment:
This has nothing to do with your code, but I just noticed that `inversed` doesn't do anything unless `filled` is also true. Maybe in the absence of better documentation or a better API we should include the same kind of "Inversed filled" example that the React spinner has.